### DISCUSSION TITLE
Add a label to let users know that an issue has been addressed.

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [mantinedev/mantine#574] [Feedback]
Does it make sense to add an `addressed` label on issues that are already addressed & would be available in the next minor release ? Currently, there is no way to tell if an issue has been resolved. 

Additionally, we could tag each issue with a milestone to let users know when the issue fix/feature would be out.

**Comment by rtivital** [comment]
Usually, issues have an associated pull request, if it does not I leave a comment when issue is resolved with release version

**Reply by UltraInstinct05** [reply]
Usually that's good, but I noticed sometimes the PR icon not appearing on issues even though the comment contains something like `Fixes #123` etc. Hence the suggestion.
